src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -304,18 +274,6 @@ def format_mri_field_value(field: str, value: str) -> str:
         if parsed_mri_field is None:
             return value
 
-        if condition_id := parsed_mri_field.mri.span_attribute_rule_id:
-            try:
-                condition = SpanAttributeExtractionRuleCondition.objects.get(id=condition_id)
-                unit = cast(MetricUnit, condition.config.unit)
-            except SpanAttributeExtractionRuleCondition.DoesNotExist:
-                with sentry_sdk.new_scope() as scope:
-                    scope.set_tag("field", field)
-                    sentry_sdk.capture_message(
-                        "Trying to format MRI field for non-existent span metric."
-                    )
-                return value
-
         else:
```

Comment:
There is no need for this `else` statement

Reply:
Removed